conn_test.go

```diff
@@ -243,6 +244,76 @@ func TestListenerAcceptFilter(t *testing.T) {
 	}
 }
 
+func TestConnReadDuringClose(t *testing.T) {
+	// Limit runtime in case of deadlocks
+	lim := test.TimeOut(time.Second * 5)
+	defer lim.Stop()
+
+	// Check for leaking routines
+	report := test.CheckRoutines(t)
+	defer report()
+
+	network, addr := getConfig()
+	listener, err := (&ListenConfig{
+		Backlog: 2,
+	}).Listen(network, addr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	msg := "foo"
+
+	conn, derr := net.DialUDP(network, nil, listener.Addr().(*net.UDPAddr))
+	if derr != nil {
+		t.Error(derr)
+	}
+	if _, werr := conn.Write([]byte(msg)); werr != nil {
+		t.Error(werr)
+	}
+
+	lConn, err := listener.Accept()
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Is it possible to simplify connection preparation part by`pipe()`?

Reply:
Yep sounds good. Tests passed locally in [5584de2](https://github.com/pion/udp/pull/80/commits/5584de262cabc7dc325674860cac6999d700acd2). Just need to wait for the workflow to run